content/en/apps/tutorials/local-setup.md

```diff
@@ -189,6 +189,7 @@ To install a valid certificate, open a terminal in the `cht-core` directory. Ens
 ```shell
 ./scripts/add-local-ip-certs-to-docker-4.x.sh cht_nginx_1
 ```
+If you are using an older version of cht-core, ensure that the  `add-local-ip-certs-to-docker.sh` file is available under the scripts folder.
```

Comment:
This line looks unrelated to the concept of an overview. Please remove it from this PR and open a separate one so we can review separately and keep things clear in the git history.